### PR TITLE
Release 4.5.0rc1

### PR DESCRIPTION
**Pull request by muhammadhamzasajjad**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by muhammadhamzasajjad** [comment]
@conda-forge-admin, please rerender

**Comment by muhammadhamzasajjad** [comment]
@conda-forge-admin, please rerender

**Comment by muhammadhamzasajjad** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml
Comment (jjerphan) [inline recipe/meta.yaml:1]: From https://github.com/conda-forge/arcticdb-feedstock/pull/233:
 - [a specification for `libboost-devel` needs to be added](https://github.com/conda-forge/arcticdb-feedstock/pull/233/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR51)
 - [a test must be skipped](https://github.com/conda-forge/arcticdb-feedstock/pull/233/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR162)

**Review by jjerphan — CHANGES_REQUESTED** [review]
Just a few changes and it LGTM.

**Comment by muhammadhamzasajjad** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml
Comment (muhammadhamzasajjad) [reply recipe/meta.yaml:1]: Done in [310d339](https://github.com/conda-forge/arcticdb-feedstock/pull/242/commits/310d3395f60bd2c069f09389111ed36bc4052ed9)

**Review by jjerphan — APPROVED** [review]
LGTM given :green_circle: checks.

Note that we can now [build with 2 threads on the feedstock](https://github.com/conda-forge/arcticdb-feedstock/pull/234/commits/e533ff442245203023b62460c1158bda7114c4cf#diff-d7075654874cb08007a21aaab3ecd4b3453a9087e7505d034d548b8938b599bcR4-R8).

**Pull request merged**